libkbfs/block_ops.go

```diff
@@ -137,6 +152,19 @@ func (b *BlockOpsStandard) Ready(ctx context.Context, md *RootMetadata,
 // Put implements the BlockOps interface for BlockOpsStandard.
 func (b *BlockOpsStandard) Put(ctx context.Context, md *RootMetadata,
 	blockPtr BlockPointer, readyBlockData ReadyBlockData) error {
+	if b.bwKBps > 0 {
+		b.lock.Lock()
+		// Simulate a constrained bserver connection
+		delay := len(readyBlockData.buf) * int(time.Second) / (b.bwKBps * 1024)
```

Comment:
Yes.  It's intended for use in tests, where there is 0 delay on the "real" connection.